Content.Server/Dice/DiceComponent.cs

```diff
@@ -24,15 +25,16 @@ public class DiceComponent : Component, IActivate, IUse, ILand, IExamine
 
         public override string Name => "Dice";
 
-        [DataField("step")]
-        private int _step = 1;
         private int _sides = 20;
-        private int _currentSide = 20;
+
         [ViewVariables]
-        [DataField("diceSoundCollection")]
-        public string _soundCollectionName = "dice";
+        [DataField("sound")]
+        private readonly SoundSpecifier _sound = new SoundCollectionSpecifier("Dice");
+
         [ViewVariables]
-        public int Step => _step;
+        [field: DataField("step")]
```

Comment:
`field:` isn't necessary anymore.

```suggestion
        [DataField("step")]
```

Reply:
I blame rider, done